### ISSUE TITLE
FTP way

### ISSUE THREAD

**Issue by arasexyz** [silexlabs/Silex#752]
Hello, i've got this message when i publish my site project 
error message 553
can't open that file, no such file or directory.

a classic error ? what's wrong i do ?

anyway thanks for this project !

**Comment by JbIPS** [comment]
Hi, what's your FTP server? Because a code `553` means `Permission denied` so I see two options:
1. You don't have write permission on the folder you want to publish
2. Your FTP server doesn't support a filename generated by Silex

In the first case, you'll have to configure your server. In the second, we'll need to find the file causing issues and understand why.

**Comment by arasexyz** [comment]
![553-erreur](https://user-images.githubusercontent.com/33373062/41199432-45062aec-6c92-11e8-95df-f1c1da22f469.png)

Hi, this screen shot after a try in firefox. same error

i'm using the ovh webhoster (start10m formula)

thanks

**Comment by JbIPS** [comment]
The error says no such file or directory. Could you show your publish settings and double check the path exists on your FTP server?

**Comment by JbIPS** [comment]
I reproduced it on my OVH storage. I'll look into that. Meanwhile, @lexoyo do you have another FTP server you can try it with? I doubt it's OVH's fault but who knows :smile:

**Comment by arasexyz** [comment]
i i've upgrade the ovh webhosting formula. thinking that it will fix something. Not a change.

**Comment by JbIPS** [comment]
Yeah, it's something weird in Silex publishing code: it looks like the base folders are not created.

As a temporary workaround, you could create `js` `css` and `assets` folders manually in you `www` folder, that should do the trick. Meanwhile, I'll look into the bug.

**Comment by JbIPS** [comment]
Actually the issue was on our FTP connector not handling OVH very well. A fix has been deployed (https://github.com/silexlabs/unifile/commit/f71112a73ba1cc2c34343d93ea20b29ba60d8b7a) and will be available in Silex in the next version. 

Let us know how the workaround went and we'll keep you in touch when the next version is out :wink:

**Comment by lexoyo** [comment]
That's a great fix @JbIPS well done!!
I'll release a new version of Silex as soon as you make it available on npm
I added a question about a console.log on your commit

@arasexyz please be patient and stay tuned. Did you try to create the 3 folders by hand as @JbIPS suggested? It will definitely work

**Comment by arasexyz** [comment]
Creating js css and assets folders manually, avoid publish error ! thx !!
